driver/src/main/scala/api/commands/DeleteCommand.scala

```diff
@@ -13,15 +13,15 @@ import reactivemongo.api.{
  */
 private[reactivemongo] trait DeleteCommand[P <: SerializationPack] { self: PackSupport[P] =>
 
-  private[api] final class Delete(
+  private[reactivemongo] final class Delete(
```

Comment:
Revert

Reply:
It's needed for tests implementation. See `driver/src/test/scala/DeleteCommandSpec.scala:52`. 

This implementation was done in analogue with already existed `driver/src/test/scala/UpdateCommandSpec.scala`. In this spec `UpdateCommand#Update` is used and this is also defined as `private[reactivemongo]`.

If it is not a good pattern for testing, @cchantep pls, advise another solution.